sql-cloudant/src/main/scala/org/apache/bahir/cloudant/CloudantChangesConfig.scala

```diff
@@ -67,7 +67,8 @@ class CloudantChangesConfig(protocol: String, host: String, dbName: String,
   }
 
   def getChangesReceiverUrl: String = {
-    var url = dbUrl + "/" + defaultIndex + "?include_docs=true&feed=continuous&timeout=" + timeout
+    var url = dbUrl + "/" + defaultIndex + "?include_docs=true&feed=normal" +
+      "&seq_interval=10000&timeout=" + timeout
```

Comment:
WDYT about making the `seq_interval` the `bulkSize` instead of hard-coding?

Reply:
Yes, I think that's a good idea.  Fixed in f758996.